Polygon_mesh_processing/doc/Polygon_mesh_processing/Polygon_mesh_processing.txt

```diff
@@ -152,6 +152,31 @@ An example of how to remesh a given triangulated surface mesh with the Delaunay
 algorithm while preserving the detected sharp edges
 is given in \ref Polygon_mesh_processing/delaunay_remeshing_example.cpp
 
+\paragraph Decimate Remeshing of (Almost) Planar Patches
+If a triangulated surface mesh uses a lot of triangles to describe some planar regions of a model, it is possible to simplify
+those parts using the function `CGAL::Polygon_mesh_processing::remesh_planar_patches()`.
+For simply connected patches, it is even possible to not retriangulate them.
+The coplanarity and collinearty tests being exact, almost coplanar patches will not be detected by default. It is possible to specify a
+threshold on the angle between adjacent faces (resp. segments) so that they are considered coplanar (resp. collinear). However, this
+coplanarity and collinearity tolerance is only local and there is no global control (think of the classical example of a circle arc
+densely sampled: 3 consecutive points are seen as almost collinear while all the points on the arc are not). To circumvent this situation,
+we also provide the function `CGAL::Polygon_mesh_processing::remesh_almost_planar_patches()` that expects the segmentation into
+planar patches and corners to be provided. The function `CGAL::Polygon_mesh_processing::region_growing_of_planes_on_faces()`
+uses the region growing algorithm to detect planar regions in a mesh with global and local criteria. Similarly, the function
+`CGAL::Polygon_mesh_processing::detect_corners_of_regions()` can be used to detect corner vertices on the border of the planar
+regions detected by running the region growing algorithm on border segments of the patch.
+
+
+\cgalFigureBegin{decimate_cheese, decimate_cheese.png, decimate_colors.png}
+Remeshing of planar patches in two models. On the right, the output has not been retriangulated and the patch ids
+have been assigned to the input and the output meshes, allowing an identical color scheme.
```

Comment:
Keep the same model for both illustrations?

Reply:
too many patches on the cheese and the color wheel of the demo being not great makes that interesting patches have almost identical colors (while being different patches)